frontend/packages/metal3-plugin/src/components/host-menu-actions.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { asAccessReview, KebabOption } from '@console/internal/components/utils';
+import { k8sKill, K8sKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { getMachineNodeName, getName } from '@console/shared';
+import { confirmModal } from '@console/internal/components/modals';
+import { NodeMaintenanceModel } from '@console/kubevirt-plugin/src/models';
+import { findNodeMaintenance, getHostMachine, getNodeMaintenanceReason } from '../selectors';
+import { startNodeMaintenanceModal } from './modals/start-node-maintenance-modal';
+
+type ActionArgs = {
+  nodeName?: string;
+  nodeMaintenance?: K8sResourceKind;
+  hasNodeMaintenanceCRD: boolean;
+};
+
+export const SetNodeMaintanance = (
+  kindObj: K8sKind,
+  host: K8sResourceKind,
+  resources: null,
+  { hasNodeMaintenanceCRD, nodeMaintenance, nodeName }: ActionArgs,
+): KebabOption => ({
+  hidden: !hasNodeMaintenanceCRD || !!nodeMaintenance,
+  label: 'Start Maintenance',
+  callback: () => startNodeMaintenanceModal({ nodeName }),
+  accessReview: asAccessReview(NodeMaintenanceModel, null, 'create'),
```

Comment:
Don't use `asAccessReview` here. That is meant for checking access on a specific object. If we really want a helper for create actions, we should create a separate helper for that purpose.

Reply:
I see, removing..